### PR TITLE
new task post route

### PR DESCRIPTION
**Pull request by ljelias**

### CODE REVIEW

File: src/tasks/tasks-router.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  taskCreator: task.creator_id,
+  taskDue: task.date_due,
+  personAssigned: task.user_assigned_id,
+  groopId: task.groop_id
+});
+
+tasksRouter.post('/:groupId', jsonParser, async (req, res, next) => {
+  const { name, description, creator_id, due_date, groop_id} = req.body;
+
+  for (const field of ['taskName', 'taskDescription', 'taskCreator', 'taskDue', 'groopId'])
```
Comment (quasarwei) [inline src/tasks/tasks-router.js]: the fields should match the fields in migrations (name, description, creator_id, date_due, groop_id). Also, the  'groop' in groop_id is bothering me, can we change it to 'group'?

Comment (ljelias) [reply src/tasks/tasks-router.js]: yes, and yes!

File: src/tasks/tasks-service.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const TasksService = {
+  postNewTask(knex, newTopicInfo) {
```
[the remark this answers is not in the document]
Comment (ljelias) [reply src/tasks/tasks-service.js]: Yes, a careless mistake. My brain was getting mushy by the end of the day.

[the remark this answers is not in the document]
Comment (ljelias) [reply src/tasks/tasks-service.js]: I can fix all in the morning, or feel free to do it if you want.

**Pull request closed without merging**